src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -52,6 +54,9 @@ public class JobService {
   private static final Logger logger = LoggerFactory.getLogger(JobService.class);
   private static final int MIN_SHUTDOWN_TIMEOUT = 14;
   private static final int POD_LISTENER_SHUTDOWN_TIMEOUT = 2;
+  // Maximum number of times that the flight sql DB will be searched while enumeratin
+  private static final int MAX_FLIGHT_SEARCH_QUERIES = 10;
```

Comment:
Should these be tunable/ via configuration?

Reply:
I was debating about that.  I'm hoping that this is actually a short term thing until I make it so that queries are pushed down into the DB